Lib/Protocols/IdHL7.pas

```diff
@@ -556,6 +594,38 @@ destructor TIdHL7.Destroy;
   end;
 end;
 
+function TIdHL7.ConvertIPtoCardinal(const AStr: AnsiString): Cardinal;

```

Comment:
Is this method really necessary?  Indy already has other functions for parsing IP addresses into numeric values. For example, `IdGlobal.IPv4ToUInt32()`

Reply:
Changed as requested
https://github.com/IndySockets/Indy/pull/442/commits/7bd8d29be4574608893391b97fcb794d959042b9